### PR TITLE
Move OCA boilerplate to the footer

### LINKED ISSUE #269: [RFC] Structured README.md
Do we have a template for repository README.md? I see a lot of different type for README in the repos and I couldnot find a template.

I am thinking a some guidance with following sections:
1. What is the purpose of the repository
2. How to install the modules
* install Odoo => URL
* install Module => Which way (git pull, pypi, docker?)
3. How to report a bug/issue (=>URL). How to contribute (=> URL)
4. List of addons
5. Translation
6. What is the OCA

The objective is to provide some basic information to the audience arriving from Google search and with not necessary information on Odoo.
The idea would not be to recreate a full How-to in the README but only remind the basics steps to get started.

Must have:
* A README.md template that we can use for manual creation

Nice to have:
* Standard template could be almost fully automated for all repos with commands pregenerated based on the repo URL (like Streisand: https://github.com/jlund/streisand).

### PR DESCRIPTION
**Pull request by dreispt**
...but leave the "Try me on Runbot" at the top, just after the addon title.

(Hopefully) fixes #269

### CODE REVIEW

**Comment by dreispt** [comment]
Comments?
I plan a follow up PR to further simplify the content structure.

File: template/module/README.rst [hunk 1]
```diff
@@ -77,16 +70,24 @@ The development of this module has been financially supported by:
 * Company 2 name
 
 Maintainer
-----------
+===========
+
+.. ATTENTION: replace the {placeholders} below
+.. project_repo is the GithHun project name. For example "maintainer-tools"
```
Comment (pedrobaeza) [inline template/module/README.rst]: s/GitHun/GitHub

**Comment by sbidoul** [comment]
@dreispt I'm not sure. I've the feeling we should keep the status-quo for now until we have a mechanism to autogenerate the boilerplate (and maintain it in a central place), then do a big leap at once.
Otherwise we'll have this intermediary format that will make migration to a new format even more difficult.

**Comment by dreispt** [comment]
In the Issue discussion there was no consensus around that automation, nor a convincing technical solution.
So I'm assuming that the manual approach is here to stay...

File: template/module/README.rst (hunk 1 above)
Comment (dreispt) [inline template/module/README.rst]: Do you have anything against [Huns](https://duckduckgo.com/?q=hun&t=ffnt&iax=1&ia=images)?

Comment (pedrobaeza) [inline template/module/README.rst]: :rofl:

File: template/module/README.rst [hunk 2]
```diff
@@ -77,16 +70,24 @@ The development of this module has been financially supported by:
...
 
 .. image:: https://odoo-community.org/logo.png
    :alt: Odoo Community Association
    :target: https://odoo-community.org
 
 This module is maintained by the OCA.
 
-OCA, or the Odoo Community Association, is a nonprofit organization whose
 mission is to support the collaborative development of Odoo features and
```
Comment (leemannd) [inline template/module/README.rst]: With the removal of the line before, I'll add `It's` to this one

**Comment by sbidoul** [comment]
This should be superseded by #335

**Comment by dreispt** [comment]
Yes. Superseded by #335

**Pull request closed without merging**